documentation/resources/install_adoptopenjdk.md

```diff
@@ -0,0 +1,46 @@
+[back to resource list](https://github.com/sous-chefs/java#resources)
+
+# adoptopenjdk_install
+
+Introduced: v7.0.0
+
+## Actions
+
+- `:install`
+- `:remove`
+
+## Properties
+
+| Name                  | Type    | Default                                                            | Description                                         | Allowed Values                         |
+| --------------------- | ------- | ------------------------------------------------------------------ | --------------------------------------------------- | -------------------------------------- |
+| version               | String  |                                                                    | The name of the resource. java version to install   |                                        |
+| variant               | String  | `openj9`                                                           | Install falvour                                     | `hotspot` `openj9` `openj9-large-heap` |
+| java_home             | String  | Based on the variant and version                                   | Set to override the java_home                       |
+| arch                  | String  | `node['kernel']['machine']`                                        | Set to overide discovered ohai platform values      |
+| url                   | String  | `node['java']['adoptopenjdk'][version][arch][variant]['url']`      | The URL to download from                            |
+| checksum              | String  | `node['java']['adoptopenjdk'][version][arch][variant]['checksum']` | The checksum for the downloaded file                |
+| md5                   | String  |                                                                    | The MD5 hash of the downloaded file                 |
+| java_home_mode        | Integer | 0755                                                               | The permission for the Java home directory          |
```

Comment:
This should probably accept a string as well since we've suggested users set modes as strings. That way the leading 0 gets respected.

Reply:
To clarify it should probably accept *both*